docs/AboutUs.md

```diff
@@ -24,23 +24,23 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 <img src="images/justweihong.png" width="200px">
 
 [[github](http://github.com/justweihong)]
-[[portfolio](https://github.com/justweihong/tp)]
+[[portfolio](https://ay2021s1-cs2103t-w13-4.github.io/tp/team/justweihong.html)]
 
 * Role: UI
 
 ### Tiffany Chan
 
 <img src="images/nottiffchan.png" width="200px">
 
-[[github](http://github.com/nottiffchan)] [[portfolio](https://github.com/nottiffchan/tp)]
+[[github](http://github.com/nottiffchan)] [[portfolio](https://ay2021s1-cs2103t-w13-4.github.io/tp/team/nottiffchan.html)]
 
 * Role: Deliverables and Deadlines
 
 ### Vuong Hoang Long
 
 <img src="images/unclegrandpa925.png" width="200px">
 
-[[github](https://github.com/UncleGrandpa925)] [[portfolio](https://github.com/UncleGrandpa925)]
+[[github](https://github.com/UncleGrandpa925)] [[portfolio](https://ay2021s1-cs2103t-w13-4.github.io/tp/team/UncleGrandpa925.html)]
```

Comment:
I think this should be all lower-case

Reply:
Err it should be your github username right?

Reply:
github username in lowercase 😄
This is my link btw: https://github.com/AY2021S1-CS2103T-W13-4/tp/blob/master/docs/team/unclegrandpa925.md